components/dashboard/src/components/PageWithSubMenu.tsx

```diff
@@ -20,30 +23,64 @@ export interface PageWithSubMenuProps {
 }
 
 export function PageWithSubMenu(p: PageWithSubMenuProps) {
-    const location = useLocation();
     return (
         <div className="w-full">
             <Header title={p.title} subtitle={p.subtitle} tabs={p.tabs} />
-            <div className="app-container flex pt-9">
+            <div className="app-container flex md:pt-9 flex-col md:flex-row">
+                {/* TODO: extract into SubMenu component and show scrolling indicators */}
                 <div>
-                    <ul className="flex flex-col text tracking-wide text-gray-500 pt-4 lg:pt-0 w-52 space-y-2">
+                    <ul
+                        className={classNames(
+                            // Handle flipping between row and column layout
+                            "flex flex-row md:flex-col items-center",
+                            "w-full md:w-52 overflow-auto md:overflow-visible",
```

Comment:
**question**: What do you think of making the horizontal scrollbar invisible? This works ok now with the pill-like design, but could clash with the tabs.

<img width="248" alt="Screenshot 2023-03-24 at 13 06 10" src="https://user-images.githubusercontent.com/120486/227505667-ddd3acae-a473-4028-80dd-ca49c0b1658a.png">